web/src/engine/websites/PCNet.ts

```diff
@@ -0,0 +1,22 @@
+import { Tags } from '../Tags';
+import icon from './PCNet.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as FoolSlide from './decorators/FoolSlide';
+
+@FoolSlide.MangaCSS(/^http?:\/\/pcnet\.patyscans\.com/)
```

Comment:
Is a more strict regex possible e.g., `/^http?:\/\/pcnet\.patyscans\.com\/comic\/[a-zA-Z0-9]+$/`?